src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -410,6 +412,8 @@
     text: az appconfig update -g MyResourceGroup -n MyAppConfiguration --arm-auth-mode pass-through
   - name: Update an App Configuration store to set ARM authentication mode set to Pass-through and enable private network access via ARM Private Link.
     text: az appconfig update -g MyResourceGroup -n MyAppConfiguration --arm-auth-mode pass-through --enable-arm-private-network-access true
+  - name: Update an App Configuration store to set a specific key-value revision retention period (in seconds).
+    text: az appconfig update -g MyResourceGroup -n MyAppConfiguration --kv-revision-retention-period 60
```

Comment:
If I am not mistaken, the minimum is 1 hour, which is 3600s. 

@avanigupta to confirm this.

If so, then perhaps, the examples should be updated to be >= the minimum

Reply:
Yes minimum is 1 hour

Reply:
Lets use 1 day in examples